snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/components/SnomedDescriptionApiTest.java

```diff
@@ -449,6 +453,132 @@ public void updateAcceptabilityAndInactivate() {
 			.statusCode(200)
 			.body("pt.id", equalTo(ptDescriptionId));
 	}
+	
+	@Test
+	public void updateAcceptabilityWithMultipleAcceptabilityMembersForSameRefset1() {
```

Comment:
Please use a valid test method name instead of a number suffix. If this test case verifies a certain scenario then pls add it to the name and also add a comment above describing the scenario. Same applies to the other test cases.

Reply:
The scenario comments are inside the body of the test case